src/App.js

```diff
@@ -2,33 +2,11 @@ import { Routes, Route } from 'react-router-dom';
 // import { Container } from 'react-bootstrap';
```

Comment:
- [ ] Please, consider creating the tests, as required:
-- App has unit tests for several components of the app and some pure functions in Redux code.
-- App has integration tests using React Testing Library